README.md

```diff
@@ -2,15 +2,19 @@
 
 # Mina Docs
 
-## To view the docs
-
-To view this website, visit [docs.minaprotocol.com](https://docs.minaprotocol.com/).
+This Mina Docs website is [docs.minaprotocol.com](https://docs.minaprotocol.com/).
 
 ## To contribute content
 
-Click the `Suggest changes` button at the bottom left of any page on [the docs website](https://docs.minaprotocol.com/) to edit that page and submit a pull request via Github.
+Thank you for your interest! See the docs [CONTRIBUTING](contributing) guidelines.
```

Comment:
@MartinMinkov  can you teach me how to verify this link?

Reply:
thanks @MartinMinkov  I know how to test links now! 
In a PR > Files changed view, press the period (`.`) key to open github.dev where I can test the links!